tests/testthat/test-utilities-quantity.R

```diff
@@ -56,7 +56,7 @@ test_that("It returns null if the quantity by path does not exist and stopIfNotF
 })
 
 test_that("It throws an error when trying to retrieve a quantity by path that would result in multiple quantities", {
-  expect_that(getQuantity(toPathString(c("Organism", "Liver", "*")), sim), throws_error())
+  expect_error(getQuantity(toPathString(c("Organism", "Liver", "*")), sim))
```

Comment:
I think we should also be checking the exact error thrown here using the `regexp` [argument](https://testthat.r-lib.org/reference/expect_error.html).

This way, we know that the expected error occurred.

Reply:
Yes, I agree, we should try to follow this practice. Sometimes the error is thrown and the test passes, but the error is actually somewhere else and we do not really test the case.

Reply:
I added the message

Reply:
Interesting. I see pros and cons. Let's discuss